### ISSUE TITLE
Problem with wp2puput import

### ISSUE THREAD

**Issue by FlipperPA** [APSL/puput#79]
Thanks for the great work on puput. After getting the installation working for `wp2puput`, I'm running into an error on the import. 

```
(pyphilly) [vagrant@vagrant pyphilly]$ ./manage.py wp2puput peregrinesalon.wordpress.2016-11-25.xml
usage: manage.py wp2puput [-h] [--version] [-v {0,1,2,3}]
                          [--settings SETTINGS] [--pythonpath PYTHONPATH]
                          [--traceback] [--no-color] [--slug SLUG]
                          [--title TITLE]
manage.py wp2puput: error: unrecognized arguments: peregrinesalon.wordpress.2016-11-25.xml
```

I've tried several different ways, but haven't dug any deeper into the code in case someone sees something obvious I'm missing.

**Comment by marctc** [comment]
Thanks! I'll check it asap!

**Comment by marctc** [comment]
Fixed in wordpress2puput 0.3
